### DISCUSSION TITLE
GPU Working Mode missing (Solved)

### DISCUSSION THREAD

**Discussion by MaxLap** [BartoszCichecki/LenovoLegionToolkit#993] [General]
Hello, I'm posting this here in case someone encounters the same problem to maybe help.

I have a Lenovo Legion Slim 7 Gen 8 with an AMD Ryzen 7840HS and a RTX 4060. 

When I first installed LLToolkit, everything was fine and I could see the GPU Working Mode setting. 

At one point, I don't know what happened, but the GPU Working Mode setting disappeared. I could still see the state of the GPU, but no option to control. I did reboots, I looked into the BIOS and the GPU was already set to switchable. I tried toggling the setting in the BIOS to Discrete only and back to Switchable Graphics and that didn't work either (I just clicked the toggle twice, no reboot).

Finally, what did work, is that I set the BIOS to Discrete only (I don't remember the exact terminology), and rebooted like that. LLToolkit finally displayed the GPU Working Mode option!. Setting it to hybrid told be I had to reboot, which I did. The BIOS is now back to switchable graphics and LLToolkit now properly shows the GPU Working Mode setting.

I wonder if an issue would increase the chance this gets found... I never think of looking at discussions on Github...

**Comment by ozanisman** [comment]
Thank you for posting this, I have the same laptop and was experiencing the same issue. Your BIOS fix also worked for me. Cheers!
